enhancements/vdpa/overview.md

```diff
@@ -0,0 +1,160 @@
+
+For additional details on this use case see [How vDPA can help network service providers simplify CNF/VNF certification](https://www.redhat.com/en/blog/how-vdpa-can-help-network-service-providers-simplify-cnfvnf-certification).
+
+#### **Use case #2: Hyperscale**
+
+Hyperscale is a use case where storage, networking and security are offloaded to smartNICs (programmable accelerators) to free up the host server's resources to run workloads such as VMs and containers. For granting these workloads the "look & feel" that they are running as usual on the host server while in practice their storage, networking and security is running on a PCI connected smartNIC, resource mapping from the host server to the smartNIC is required. 
+
+This is where vendor specific closed solutions and in contrast virtio/vDPA technologies come into play providing the resource mapping and enabling the hyperscale use case to come to life. The key value virtio/vDPA brings on top of it being an open and standard interface is in this context a single data plane / control plane for all workloads.  This means that by using virtio/vDPA for the hyperscale use case all workloads (VMs, containers, Linux processes) can share the exact same control plane / data plane which significantly simplifies maintaining, troubleshooting and controlling resources. 
+
+For additional details on this use case see [Hyperscale virtio/vDPA introduction: One control plane to rule them all](https://www.redhat.com/en/blog/hyperscale-virtiovdpa-introduction-one-control-plane-rule-them-all).
+
+#### **Use case #3: Confidential containers and hardening**
+
+As smartNICs are emerging and their corresponding devices and SW drivers, new attack surfaces are being created in the kernel and host. This is since historically we assume devices are “dumb” while drivers are “smart” thus we focus is on protecting the device from the driver. With smartNICs however you can now run malicious software on the smartNIC transforming the device to a smart entity that can attack the driver. This is especially critical when looking at emerging use cases such as confidential computing/confidential containers which also require using devices and drivers. Virtio/vDPA addresses this problem by hardening the open standard framework in the kernel and it’s corresponding drivers while closed source solutions are much more vulnerable to such attacks (virtio has a large community supporting it). 
+ 
+For additional details on this use case see [Hardening Virtio for emerging security use cases](https://www.redhat.com/en/blog/hardening-virtio-emerging-security-usecases).
+
+### Goals
+
+- As a user I want to configure an Openshift cluster with VDPA support
+
+### Non-Goals
+
+
+### Supported Hardware
+
+We want to support the following NVIDIA NIC cards:
+- CTX-6 DX
```

Comment:
I think it is better to specify this like so: ConnectX-6 Dx . This is more consistent with NVIDIA documentation.

Reply:
will do